src/wasm-lib/kcl/src/lib.rs

```diff
@@ -8,11 +8,54 @@
 #[allow(unused_macros)]
 macro_rules! println {
     ($($rest:tt)*) => {
+        #[cfg(feature = "disable-println")]
+        {
+            let _ = format!($($rest)*);
```

Comment:
Why add this? If there are side-effects in the args to format?

Reply:
Yes, and also to avoid dead code warnings when building with disable-println